<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dturulefilewidgetClass</class>
 <widget class="QWidget" name="dturulefilewidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dturulefilewidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>规约历史文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件类型:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>SOE记录</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>遥控记录</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>日志</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>定点文件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>极值文件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>自检信息</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>清空所有规约历史文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>dturulefilewidgetClass</receiver>
   <slot>showfile(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dturulefilewidgetClass</receiver>
   <slot>updatelist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatelist()</slot>
  <slot>showfile(int,int)</slot>
 </slots>
</ui>
